fenix/app/src/main/java/org/mozilla/fenix/compose/tabstray/TabListItem.kt

```diff
@@ -89,37 +91,43 @@ fun TabListItem(
 
         Column(
             modifier = Modifier
-                .padding(horizontal = 16.dp)
+                .padding(start = 12.dp)
                 .weight(weight = 1f),
         ) {
             Text(
                 text = tab.toDisplayTitle().take(MAX_URI_LENGTH),
+                color = FirefoxTheme.colors.textPrimary,
                 fontSize = 16.sp,
+                letterSpacing = 0.0.sp,
+                overflow = TextOverflow.Ellipsis,
                 maxLines = 2,
-                color = FirefoxTheme.colors.textPrimary,
             )
 
             Text(
                 text = tab.content.url.toShortUrl(),
-                fontSize = 12.sp,
                 color = FirefoxTheme.colors.textSecondary,
+                fontSize = 14.sp,
+                letterSpacing = 0.0.sp,
+                overflow = TextOverflow.Ellipsis,
+                maxLines = 1,
             )
         }
 
-        if (!multiSelectionEnabled) {
-            IconButton(
-                onClick = { onCloseClick(tab) },
-                modifier = Modifier.size(size = 24.dp),
-            ) {
-                Icon(
-                    painter = painterResource(id = R.drawable.mozac_ic_close),
-                    contentDescription = stringResource(
-                        id = R.string.close_tab_title,
-                        tab.content.title,
-                    ),
-                    tint = FirefoxTheme.colors.iconPrimary,
-                )
-            }
```

Comment:
why was this `if` removed?

Reply:
I moved the parameter into the aplha calculation. 

We want to keep the text positioned the same when we hide the icon. If we don't draw the icon, the text takes up more space.

Reply:
That's exactly what I was trying to avoid, I really don't like having two blocks with different views being initiated.

We could also put the condition into the onClick event.

```
IconButton(
            onClick = { if (!multiSelectionEnabled) onCloseClick(tab) },
            modifier = Modifier
                .alpha(if (!multiSelectionEnabled) 1f else 0f)
                .size(size = 48.dp)
                .padding(12.dp),
        ) {
            Icon(
                painter = painterResource(id = R.drawable.mozac_ic_close),
                contentDescription = stringResource(
                    id = R.string.close_tab_title,
                    tab.content.title,
                ),
                tint = FirefoxTheme.colors.iconPrimary,
            )
        }
```

Reply:
I see! Updated :)